scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -721,13 +737,89 @@ def create_cloudsql_instance(self):
 
     def generate_deployment_templates(self):
         """Generate deployment templates."""
-        # TODO: generate deployment templates
-        # TODO: ask if we should create the deployment
-        pass
+        self._print_banner('Generate Deployment Manager templates')
+
+        deploy_tpl_path = os.path.abspath(
+            os.path.join(
+                self.DEPLOY_TPL_DIR_PATH,
+                'deploy-forseti.yaml.in'))
+        out_tpl_path = os.path.abspath(
+            os.path.join(
+                self.DEPLOY_TPL_DIR_PATH,
+                'deploy-forseti-{}.yaml'.format(os.getpid())))
+
+        # Ask for SendGrid API Key
+        sendgrid_api_key = raw_input('What is your SendGrid API key? ').strip()
+        if sendgrid_api_key:
+            self.sendgrid_api_key = sendgrid_api_key
+
+        # Ask for notification sender email
+        notif_sender_email = raw_input(
+            'What is the sender email address for your notifications? '
+            '(press [enter] to leave blank) ').strip()
+        if notif_sender_email:
+            self.notification_sender_email = notif_sender_email
+
+        # Ask for notification recipient email
+        notif_recip_email = raw_input(
```

Comment:
These abbreviations look a bit odd to read.  If you think the full spellings are too long, then how about just using ```recipient_email```?  Ditto for ```notif_sender_email``` above.